rcl_lifecycle/src/transition_map.c

```diff
@@ -42,6 +42,8 @@ rcl_ret_t
 rcl_lifecycle_transition_map_is_initialized(const rcl_lifecycle_transition_map_t * transition_map)
 {
   rcl_ret_t is_initialized = RCL_RET_OK;
+  RCL_CHECK_FOR_NULL_WITH_MSG(
+    transition_map, "transition_map pointer is null\n", return RCL_RET_INVALID_ARGUMENT);
```

Comment:
you are checking here twice the `transition_map`. Can you use the same `RCL_CHECK_FOR_NULL_WITH_MSG` for `transition_map->transitions`?

Reply:
Do you mean do like this?
```diff
 diff --git a/rcl_lifecycle/src/transition_map.c b/rcl_lifecycle/src/transition_map.c
index 900683c..55ee028 100644
--- a/rcl_lifecycle/src/transition_map.c
+++ b/rcl_lifecycle/src/transition_map.c
@@ -41,13 +41,14 @@ rcl_lifecycle_get_zero_initialized_transition_map()
 rcl_ret_t
 rcl_lifecycle_transition_map_is_initialized(const rcl_lifecycle_transition_map_t * transition_map)
 {
-  rcl_ret_t is_initialized = RCL_RET_OK;
   RCL_CHECK_FOR_NULL_WITH_MSG(
     transition_map, "transition_map pointer is null\n", return RCL_RET_INVALID_ARGUMENT);
-  if (!transition_map->states && !transition_map->transitions) {
-    is_initialized = RCL_RET_ERROR;
-  }
-  return is_initialized;
+  RCL_CHECK_FOR_NULL_WITH_MSG(
+    transition_map->states, "transition_map->states is null\n", return RCL_RET_INVALID_ARGUMENT);
+  RCL_CHECK_FOR_NULL_WITH_MSG(
+    transition_map->transitions, "transition_map->transitions is null\n",
+    return RCL_RET_ERROR);
+  return RCL_RET_OK;
 }

 rcl_ret_t

```
If so, the code logic may changed and the test case will failed.

Reply:
@ahcorde 
Any tips?